pages/index.tsx

```diff
@@ -53,9 +53,12 @@ const Home: NextPage = () => {
             )}
           </div>
         </Row>
+        <Row title='Also interested in' link={{name: 'See all Journal posts', url: '/journal'}}>
+
```

Comment:
Find a way to filder throw the categories. 

like the prototype on [figma](https://www.figma.com/file/hMqjNAis2GH3TAYezwERIt/Next-porfolio?node-id=6%3A3)